apps/core/src/utils/analytics/persistableStorage.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I put this under an `/analytics` folder because I'm going to add another general utility here for Sui Explorer and our other web applications